### PR TITLE
[AB#27822] Remove Accept-CRS requirement stub

### PR DESCRIPTION
**Pull request by lbam**
This isn't doing anything and is making refactoring harder.

### CODE REVIEW

File: src/rest_framework_dso/views.py [hunk 1]
```diff
@@ -327,21 +320,17 @@ def initial(self, request, *args, **kwargs):
...
         Clients provide this header to indicate which CRS
         they would like to have in the response.
         """
-        if not http_value:
-            if self.accept_crs_required:
-                # This makes Accept-Crs mandatory
-                raise PreconditionFailed("The HTTP Accept-Crs header is required")
-            else:
-                return None
+        if value is None:
```
[the remark this answers is not in the document]
Comment (lbam) [reply src/rest_framework_dso/views.py:329]: The point is not the exception. `accept_crs_required` is always False, so this is never reached.

[the remark this answers is not in the document]
Comment (jjmurre) [reply src/rest_framework_dso/views.py:329]: Ah, I see what you mean. However, maybe the idea is to be able to use this `DSOViewMixin` in your class and then change the setting of `accept_crs_required`? These overridable-class-level-variables approach is used quite often in Django Rest Framework.

[the remark this answers is not in the document]
Comment (lbam) [reply src/rest_framework_dso/views.py:329]: I guess (you wrote this code, if `git blame` is right), but we never change this and nothing in DRF changes it either.

[the remark this answers is not in the document]
Comment (jjmurre) [reply src/rest_framework_dso/views.py:329]: If it's never used, we can just trash it. I vaguely remember that this header has something to do with the DSO standard. I'll ping him about it.

[the remark this answers is not in the document]
Comment (jjmurre) [reply src/rest_framework_dso/views.py:329]: Diederik also could not recall that this header should possibly be required, let's dump it, KISS.

**Review by jjmurre — APPROVED** [review]
LGTM

**Pull request merged**